.github/workflows/upload-frontend-secure-pipelines.yml

```
@@ -35,35 +35,15 @@ jobs:
         with:
           name: self-service-front
 
-      - name: Assume AWS Role
-        uses: aws-actions/configure-aws-credentials@v1-node16
-        with:
-          role-to-assume: ${{ secrets.FRONTEND_BUILD_GHA_ROLE_ARN }}
-          aws-region: eu-west-2
-
-      - name: Push Docker image
-        id: push-docker-image
-        uses: alphagov/di-github-actions/aws/ecr/build-docker-image@5dacba941def444d69f02843feaebed79b5fca51
-        with:
-          repository: ${{ secrets.FRONTEND_BUILD_ECR_REPOSITORY }}
-          image-version: ${{ github.sha }}
-          image-tags: ${{ github.head_ref || github.ref_name }}
-          dockerfile-path: infrastructure/frontend/Dockerfile
-
-      - name: Validate template
-        run: |
-          sam validate --template "${TEMPLATE}"
-          sam validate --template "${TEMPLATE}" --lint
-
-      - name: Update template with ECR image tag
-        env:
-          ECR_REGISTRY: ${{ secrets.FRONTEND_BUILD_ECR_REGISTRY }}
-          ECR_REPOSITORY: ${{ secrets.FRONTEND_BUILD_ECR_REPOSITORY }}
-        run: sed -i "s|CONTAINER-IMAGE-PLACEHOLDER|$ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA|" "${TEMPLATE}"
-
-      - name: Upload SAM artifact
-        uses: alphagov/di-devplatform-upload-action@v3.2
+      - name: Deploy SAM app to ECR
+        uses: alphagov/di-devplatform-upload-action-ecr@d4be332eb73f1e19c0ffecc1213c6cfdae0240a4
```

```suggestion
        uses: alphagov/di-devplatform-upload-action-ecr@...
```
Use the latest released tag rather than a SHA

---

fixed